components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/SAMLSSOConfigServiceImpl.java

```diff
@@ -96,6 +97,23 @@ public SAMLSSOServiceProviderDTO createServiceProvider(SAMLSSOServiceProviderDTO
 
         try {
             SAMLSSOConfigAdmin configAdmin = new SAMLSSOConfigAdmin(getConfigSystemRegistry());
+            if (!StringUtils.isNotBlank(spDto.getSigningAlgorithmURI())
```

Comment:
can't we use StringUtils.isBlank instead of double negation?

Reply:
check other scenarios as well

Reply:
addressed in 4218e66f5163332a0be366598fd467c19187e8a5